src/Microsoft.TemplateEngine.TemplateLocalizer/xlf/LocalizableStrings.cs.xlf

```diff
@@ -29,12 +29,12 @@
       </trans-unit>
       <trans-unit id="command_export_log_cancelled">
         <source>"export" command for the following file was cancelled: "{0}".</source>
-        <target state="needs-review-translation">Příkaz „--exportovat“ byl u následujícího souboru zrušen: „{0}“.</target>
+        <target state="new">"export" command for the following file was cancelled: "{0}".</target>
```

Comment:
@cristianosuzuki77 Is this an expected change?
This string was translated before. Then we removed `--` from it. The `state` was correctly set to `needs-review-translation` afterwards.

But now we get this PR, reverting the whole text to `new` state. Shouldn't the existing translation be kept until it is corrected?